Tasks/IISWebAppManagementOnMachineGroup/IISWebAppManagementOnMachineGroup.ps1

```diff
@@ -78,6 +79,12 @@ $appPoolPassword = Get-VstsInput -Name "AppPoolPassword"
 # Additional appcmd inputs 
 $appCmdCommands = Get-VstsInput -Name "AppCmdCommands"
 
+if($enableIIS -eq "true")
+{
+    import-module servermanager
+    add-windowsfeature web-server -includeallsubfeature
```

Comment:
Is there a way to show progress? Can you invoke this command in async manner?

Reply:
Show Progress will be a tricky task because action is being performed on VM as a single stage single command execution. Enabling such progress bar require changes in Add-WindowsFeatures cmdlet itself.
It should be a sync call only as rest of the code is expecting IIS as a prerequisite.

Reply:
add a log